transport/ws.extension/pom.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
That's the way I did it to start with, but I run into a circular dependency when I did the resource injection to GatewayContextResolver. I could not make server module depend on transport/ws because that created a circular dependency (since it depends on transport.http which in turn server depends on). Thinking about it again I am not quite sure why that was an issue so I can try gain if you want.